src/components/todos/TodoItem.js

```diff
@@ -4,10 +4,45 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faTrashAlt } from '@fortawesome/free-solid-svg-icons';
 import './TodoItem.scss';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
 
 class TodoItem extends Component {
-  state = {
-    isClicked: false,
+
+  constructor(props){
+    super(props);
+    this.state = {
+      isClicked: false,
+      todoTitle: props.todo.todoItem.title,
+    }
+  }
+
+  componentDidMount() {
+    this.updateTodoTitle();
```

Comment:
You don't want to make a call to update the todo item every time one of these components is mounted, you don't have a value to update the todo item at this point yet.